pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarColumnMetadata.java

```diff
@@ -88,8 +85,6 @@ public String toString() {
         return "PulsarColumnMetadata{"
             + "isInternal=" + isInternal
             + ", nameWithCase='" + nameWithCase + '\''
-            + ", fieldNames=" + Arrays.toString(fieldNames)
-            + ", positionIndices=" + Arrays.toString(positionIndices)
```

Comment:
Does the `DecoderExtraInfo` need to be printed?

Reply:
@gaoran10 thanks for the remind,  fixed as suggestion.